polaris/hub/polarisfs.py

```diff
@@ -1,6 +1,9 @@
 from typing import TYPE_CHECKING, Any, Dict, List, Optional, Union
-
+import datetime
 import fsspec
+import hashlib
+import time
```

Comment:
This is an unused import, because of which `ruff` fails. 

You can check the linting locally with: 
```
ruff check .
```

You can also see the CICD logs in the GH actions. For this PR, you can find these [here](https://github.com/polaris-hub/polaris/actions/runs/8177921121/job/22360677551?pr=73).